packages/application-components/src/components/dialogs/confirmation-dialog/README.md

```diff
@@ -35,7 +35,7 @@ Confirmation dialogs are controlled components used to prompt the user to confir
 | `isOpen`                        | `boolean`                  |    ✅    | -                 | -                                 | Indicates whether the dialog is open or closed. The parent component needs to manage this state                                                                                                                                 |
 | `onClose`                       | `function`                 |    -     | -                 | -                                 | Called when the dialog closes (click on overlay, click on close button, press ESC). If the function is not provided, the modal cannot be closed by any of the listed options.                                                   |
 | `children`                      | `node`                     |    ✅    | -                 | -                                 | Content rendered within the dialog. If the content is long in height (depending on the screen size) a scrollbar will appear                                                                                                     |
-| `horizontalConstraint`          | `string`                   |          | `m`, `l`, `scale` | `l`                               | Horizontal width limit of the dialog card                                                                                                                                                                                       |
+| `size`                          | `string`                   |          | `m`, `l`, `scale` | `l`                               | Horizontal width limit of the dialog card                                                                                                                                                                                       |
```

Comment:
The `Constraints.Horizontal` component actually sets a `max-width`, but for the modal sizes we need an actual `width`, hence this change